gradle.properties

```diff
@@ -1,22 +0,0 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
-pockethub_github_client=dummy_client
-pockethub_github_secret=dummy_secret
-pockethub_github_callback=http://dummy.example.com
```

Comment:
It should be ignored, in my opinion.

It's a hassle to intentionally keep this file from a commit because we change its content with our credentials.
It's even more hassle when it's accidentally staged in a commit, and realize this in a last minute.
And finally, people might be accidentally pushing their credentials in their PR.

Let's prevent that in the first place.

This file should NOT be present in the repo. 
After all, the [setup environment](https://github.com/pockethub/PocketHub#setup-environment) is clear enough that you need to create local `gradle.properties` with some specific credential properties in it.